cmd/notary/keys.go

```diff
@@ -33,7 +33,7 @@ var cmdKeyListTemplate = usageTemplate{
 }
 
 var cmdRotateKeyTemplate = usageTemplate{
-	Use:   "rotate [ GUN ] [ key role ]",
+	Use:   "rotate [ GUN ] [ key role ] [ key ]",
```

Comment:
Since `key` is a flag and not a positional argument, we shouldn't need to include it on this part of the usage.  It will print with a `[flags]` appended to it